src/main/webapp/jsp/studentHome.jsp

```diff
@@ -7,7 +7,7 @@
 <c:set var="jsIncludes">
   <script type="text/javascript" src="/js/studentHome.js"></script>
 </c:set>
-<ts:studentPage pageTitle="TEAMMATES - Student" bodyTitle="Student Home" jsIncludes="${jsIncludes}">
+<ts:studentPage pageTitle="Home" bodyTitle="Student Home" jsIncludes="${jsIncludes}">
```

Comment:
@whipermr5 I think we should be consistent over here. In instructors home page we are using `Home [Instructor]` and here we are using `Student Home` :thinking: